packages/ui/src/composables/useFormComponent.ts

```diff
@@ -125,6 +118,19 @@ export function useFormComponent (props: Record<string, any>, context: any) {
     },
   })
 
+  const formContext = {
+    resetValidation,
+    validate,
+    reset,
+    hasError,
+    focus,
+    blur,
+  }
+
+  onMounted(() => formProvider?.onChildMounted?.(formContext))
```

Comment:
Double check if we need both hooks useValidation and useFormComponent after this fix, please.

Reply:
In this hotfix, I will leave both hooks. But they do the same, so we need to choose one of them.